docs/source/technical_tutorials/working_with_r/cmr_search_in_r.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>I think it could be helpful to drop a link to the collection discovery UI here because there is a chance that tool would be easier/simpler to use than going through the API using R! <a href="https://discover.maap-project.org" target="_blank">https://discover.maap-project.org</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>